app/models/alchemy/element.rb

```diff
@@ -107,20 +144,32 @@ def create_from_scratch(attributes)
       #
       # == Example
       #
-      #   @copy = Alchemy::Element.copy(@element, {:public => false})
+      #   @copy = Alchemy::Element.copy(@element, {public: false})
       #   @copy.public? # => false
       #
-      def copy(source, differences = {})
-        source.attributes.stringify_keys!
+      def copy(source_element, differences = {})
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for copy is too high. [15.26/15]